datasets/noaa-mrms-qpe/collection/noaa-mrms-qpe-1h-pass1/template.json

```diff
@@ -0,0 +1,186 @@
+{
+    "stac_version": "1.0.0",
+    "type": "Collection",
+    "id": "noaa-mrms-qpe-1h-pass1",
+    "title": "NOAA MRMS QPE 1-Hour Pass 1",
+    "description": "{{ collection.description }}",
+    "license": "proprietary",
+    "links": [
+        {
+            "rel": "license",
+            "href": "https://www.nssl.noaa.gov/projects/mrms/nmq_data_policy_OGCrevised.pdf",
+            "type": "application/pdf",
+            "title": "MRMS Dataset Sharing Policy"
+        },
+        {
+            "rel": "about",
+            "href": "https://mrms.nssl.noaa.gov",
+            "type": "text/html",
+            "title": "MRMS Homepage"
+        },
+        {
+            "rel": "about",
+            "href": "https://vlab.noaa.gov/web/wdtd/-/multi-sensor-qpe-1?selectedFolder=9234881",
+            "type": "text/html",
+            "title": "MRMS QPE Technical Product Guide"
+        }
+    ],
+    "stac_extensions": [
+        "https://raw.githubusercontent.com/stactools-packages/noaa-mrms-qpe/main/extension/schema.json",
```

Comment:
Actually, this is complicated. I can't insert the `v0.2.0` into the template and the Items `stac_extensions` list because the schema checks for an href in that list that contains `main` rather than `v0.2.0`, and so validation fails. As currently organized, the schema is built to point to the `main` branch. Not good. 

The custom schema is nice, but not necessary. I'm eliminating it to remove this problem.